org.mixedrealitytoolkit.input/InteractionModes/INearInteractionModeTarget.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// This interface is used to update the enable state of the Components that are in the <see cref="ProximityEnabledComponents"/>
+    /// array (set in Editor) and to keep track of which <see cref="Collider"/> and <see cref="XRBaseInteractor"/> duples are triggering proximity.
+    /// </summary>
+    /// <remarks>
+    /// The interface is needed to prevent a circular reference between MRTK Input and MRTK UX Core Scripts packages.
+    /// </remarks>
+    public interface INearInteractionModeTarget
+    {
+        /// <summary>
+        /// Registers the duple Collider + XRBaseInteractor as triggering proximity.
+        /// </summary>
+        /// <param name="collider">Collider triggering proximity.</param>
+        /// <param name="xrBaseInteractor">Interactor triggering proximity.</param>
+        public void RegisterActiveColliderWithInteractor(Collider collider, XRBaseInteractor xrBaseInteractor);
```

Comment:
Updated in latest commit.  Note: I actually used `ProximityEnteredEventArgs` and `ProximityExitedEventArgs` instead of `ProximityEnteredArgs` and `ProximityExitedArgs` because the first names are more similar to the existing code.  Resolving this conversation.